script/WormholeL2FlexibleVotingDeploy.s.sol

```diff
@@ -115,4 +137,46 @@ contract WormholeL2FlexibleVotingDeploy is Script, ScriptConstants {
     vm.broadcast();
     l1TokenBridge.initialize(address(l2Token));
   }
+
+  function _setupGovernor() internal returns (address, address, bool) {
+    address governorAddress = vm.envOr("L1_GOVERNOR_ADDRESS", address(0));
+    address l1TokenAddress = vm.envOr("L1_TOKEN_ADDRESS", address(0));
+    bool compToken = vm.envOr("L1_COMP_TOKEN", false);
```

Comment:
nit: prefer `isCompToken` naming for bools

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/ea5da721be6dca05841003f7ca43335e1dd662ea